.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: 🚀 Feature Request
+description: Suggest an idea for this project
+title: "short issue description"
+labels: [feature-request, needs-triage]
+assignees: []
+body:
```

Comment:
given how simple this is, do we need both Describe the feature and also Use Case?

Reply:
I think the use case section is often a section where customers feel compelled to share info they otherwise might not have shared that's helpful. Lots of times I think you're right that this isn't the case but it can sometimes not be immediately clear why some feature is desired, and so the customer explaining a use case for the feature helps clear things up.